gnd/src/main/java/com/google/android/gnd/MainActivity.java

```diff
@@ -45,6 +44,7 @@
 import dagger.android.support.DaggerAppCompatActivity;
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import timber.log.Timber;
```

Comment:
nit: These logging changes would be better in a different PR since they are unrelated to the DataBinding change.